lib/css/components/_stacks-topbar.less

```diff
@@ -0,0 +1,366 @@
+    --theme-topbar-search-background:          lighten(@topbar-actual-background, @topbar-search-lightness-increase);
+    --theme-topbar-search-placeholder:         lighten(@topbar-actual-background, 60% + @topbar-search-lightness-increase);
+    --theme-topbar-search-border:              lighten(@topbar-actual-background, @topbar-search-lightness-increase);
+    --theme-topbar-search-border-focus:        lighten(@topbar-actual-background, 45% + @topbar-search-lightness-increase);
+    --theme-topbar-search-shadow-focus:        0 0 0 @su4 fade(@white, 30%);
+
+    // Search switcher
+    --theme-topbar-select-color:               lighten(@topbar-actual-background, 60% + @topbar-search-lightness-increase);
+    --theme-topbar-select-background:          lighten(@topbar-actual-background, 10% + @topbar-search-lightness-increase);
+
+    // topbar items
+    --theme-topbar-item-color:                 @black-025;
+    --theme-topbar-item-color-hover:           @white;
+    --theme-topbar-item-background-hover:      @black-700;
+    --theme-topbar-item-color-current:         @white;
+    --theme-topbar-item-border-current:        var(--theme-primary-color);
+
+    // themed border accent
+    --theme-topbar-accent-border:              none;
+}
+
+.s-topbar {
+    min-width: auto;
+    box-shadow: var(--bs-sm);
+    width: 100%;
+    z-index: @zi-navigation-fixed;
+    background-color: var(--theme-topbar-background-color);
+    height: var(--topbar-height);
+    display: flex;
+    border-top: var(--theme-topbar-accent-border);
+    position: relative;
+
+    // TODO necessary? used to wrap content so the topbar stretches 100% w/ content at some value below that
+    .s-topbar--container {
+        width: 100%; // TODO note - consumers should use atomic classes to override this
+        display: flex;
+        margin: 0 auto;
+    }
+
+    .s-topbar--logo {
+        padding: 0 @su8;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        background-color: transparent;
+    }
+
+    a.s-topbar--logo:hover {
+        background-color: var(--theme-topbar-item-background-hover);
+    }
+
+    a.s-topbar--logo.is-selected {
+        background-color: var(--theme-topbar-item-background-hover);
+    }
+
+    .s-topbar--menu {
+        display: flex;// none;
+        padding: 0;
+        height: 100%;
+        width: 44px;
+        flex-shrink: 0;
+        align-items: center;
+        justify-content: center;
+
+        // NOTE: taken from source file
+        // build hamburger icon manually instead of using `@Svg` helper because
+        // we can more easily animate its content (transform hamburger into x)
+        span {
+            width: 18px;
+            height: 2px;
+            background-color: var(--theme-topbar-item-color);
+            position: relative;
+
+            &:before,
+            &:after {
+                position: absolute;
+                content: '';
+                width: 18px;
```

Comment:
We should put this down to `@su16` for more consistent rendering with `@Svg.Hamburger` but we'll need to double check all the rotation math.

Reply:
No math changes needed, thankfully!